src/app/common/stores/applications/SsvWeb/Wallet.store.ts

```diff
@@ -209,7 +270,7 @@ class WalletStore extends BaseStore implements Wallet {
     const myAccountStore: MyAccountStore = this.getStore('MyAccount');
     const applicationStore: Application = this.getStore('Application');
     this.accountAddress = '';
-    this.onboardSdk.walletReset();
+    // this.onboardSdk.walletReset();
```

Comment:
Why it was needed wallet reset before and now doesn't?

Reply:
this function was used when We disconnect from our wallet but in new onboard we don't have this function and also don't need it